docs/welcome/benchmarks.mdx

```diff
@@ -0,0 +1,110 @@
+---
```

Comment:
I don't like that these very technical and difficult to parse benchmark dumps are the second thing that the user sees after the welcome page. Please remove, we can just link to the Github README that says the same thing.

Reply:
yeah I feel that... We really should make some proper visual benchmarks